site/content/en/docs/Guides/feature-stages.md

```diff
@@ -45,7 +45,6 @@ The current set of `alpha` and `beta` feature gates:
 | [GameServer Stable Network ID]({{% ref "/docs/Reference/gameserver.md#stable-network-id" %}})                         | `PodHostname`                  | Enabled  | `Beta`  | 1.32.0 |
 | [Reset Metric Export on Fleet / Autoscaler deletion]({{% relref "./metrics.md#dropping-metric-labels" %}})            | `ResetMetricsOnDelete`         | Enabled  | `Beta`  | 1.32.0 |
 | [Split `agones-controller` ](https://github.com/googleforgames/agones/issues/2797)                                    | `SplitControllerAndExtensions` | Enabled  | `Beta`  | 1.32.0 |
-| [GameServer state filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239)           | `StateAllocationFilter`        | Enabled  | `Beta`  | 1.26.0 |
```

Comment:
We'll need to use a `feature` shortcode for the documentation please.

Reply:
This line has been added in the expiryVersion shortcode and removed it from the publishVersion shortcode.

Reply:
Should we also delete line 34 `| [GameServer state filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239)           | `StateAllocationFilter`        | Enabled  | `Beta`  | 1.26.0 |`?

Reply:
This line is in expiryVersion of feature shortcode and lines in expiryVersion shortcode will be removed during the release. Could you please correct me if I am wrong?